src/test/java/com/wikia/webdriver/common/testnglisteners/BrowserAndTestEventListener.java

```diff
@@ -53,7 +54,19 @@ public void afterNavigateTo(String url, WebDriver driver) {
     if (method != null) {
       Class<?> declaringClass = method.getDeclaringClass();
 
-      String cookieDomain = String.format(".%s", Configuration.getEnvType().getDomain());
+      // Fallback to wikia.com domain
+      String cookieDomain = Configuration.getEnvType().getWikiaDomain();
+
+      for(EnvType env: EnvType.values()){
+        String current_url = driver.getCurrentUrl();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)